demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/repositories/StatusesRepository.java

```diff
@@ -0,0 +1,7 @@
+package org.eclipse.tractusx.demandcapacitymgmt.demandcapacitymgmtbackend.repositories;
```

Comment:
Licence and copyright header missing. Please add.

Reply:
@carslen Done